docs/DEPLOYMENT.md

```
@@ -0,0 +1,42 @@
+- `DATABASE_URL`: Will be synced from GitHub Actions during build
+- `NODE_ENV`: Set to "production"
+
+## Local Development
+
+For local development, create a `.env` file in the project root:
+
+```
+DATABASE_URL=your_database_url_here
+```
+
+This file is already in `.gitignore` and won't be committed to the repository.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)